src/nycdb/database.py

```diff
@@ -23,25 +24,27 @@ def sql(self, SQL):
             curs.execute(SQL)
         self.conn.commit()
 
-    def insert(self, row):
-        """ Inserts one row in a transaction"""
-        SQL = sql.insert(self.table_name, row)
-        with self.conn.cursor() as curs:
-            curs.execute(SQL, row)
-        self.conn.commit()
```

Comment:
Oh, this isn't really relevant but it was the one other place where `sql.insert()` was being used, but it didn't seem to be used anywhere itself, so I removed it.